### ISSUE TITLE
Feedback UI

### ISSUE THREAD

**Issue by pvdz** [thlorenz/deoptigate#15]
Awesome tool!

As per https://twitter.com/thlorenz/status/1189921914845892613

#10 is the main blocker (v8.log contains new data that deoptigate can't deal with and results in a hard crash). This issue also has a work-around (scrubbing the v8 log) which unblocks. Would suggest to try to ignore lines the code does not understand/expect.

I can't use the cli since I use modules and need to pass on an experimental flag to node, which deoptigate won't/can't currently do. Don't think this is a hard fix, not sure if it's worth the trouble (but maybe).

I think the UI is slightly off, see blow (linux, same in firefox and chrome). Contrast is rather dark and I think that wasn't intended. Additionally some markers are not finding the right spot.

![image](https://user-images.githubusercontent.com/209817/67960084-0709f680-fbfa-11e9-88be-38ed48fc9d85.png)

When I tried to open the whole thing in chrome for the first time it would spin forever (admittedly on a 15.000 loc file) while firefox is okay. I was able to open it in chrome later could very well be size related.

**Comment by thlorenz** [comment]
I'll keep this open even though you mention that the blocker part is already tracked in #10.

Have you tried turning code highlight off?
I thought I disabled that automatically for large files, but am not sure. Not sure about the rather dark UI.
